### PR TITLE
優化簡體文檔『聲明』部分的語言表達

### PR DESCRIPTION
**Pull request by xiewei18**
主要的打磨如下：
- 對照英文原文，盡量保留原文的段落結構和句子整體結構。
- 在此基礎上，調整句子結構以與中文文法對應。
- 細節調整：將使用協助改為幫助文檔；將idot的翻譯從白癡修改為傻瓜。

### CODE REVIEW

File: README-zh_CN.md [hunk 1]
```diff
@@ -61,13 +61,13 @@ Copyleft 2001 by D.H.Grand(nOBODY/Ginux), 2010 by Gasolin, 2015 by Ryan Wu
 
 ## 声明
 
-许多项目在他们的使用协助/说明网页中链接了本指南，这么做很好，我们也鼓励大家都这么做。但如果你是负责管理这个项目网页的人，请在超链接附近的显著位置上注明：
+许多项目在他们网站的帮助文档中链接了本指南。这很好，这正是我们想要的用途。但如果你是该项目管理员并试图创建指向本指南的超链接，请在超链接附近的显著位置注明：
```
Comment (xiewei18) [inline README-zh_CN.md:64]: > That's fine, it's the use we intended.

翻譯為「正是我們想要的用途」比「鼓勵大家都這麽做」更符合原文表達一些。

File: README-zh_CN.md [hunk 2]
```diff
@@ -61,13 +61,13 @@ Copyleft 2001 by D.H.Grand(nOBODY/Ginux), 2010 by Gasolin, 2015 by Ryan Wu
...
 
 **本指南不提供此项目的实际支持服务！**
 
-我们已经深刻领教到少了上述声明所带来的痛苦。因为少了这点声明，我们不停地被一些白痴纠缠。这些白痴认为既然我们发布了这本指南，那么我们就有责任解决世上所有的技术问题。
+我们已经深刻领教到缺少上述声明所带来的痛苦：我们将不停地被那些认为发布这本指南就意味着有责任解决世上所有技术问题的傻瓜苦苦纠缠。
```
Comment (xiewei18) [inline README-zh_CN.md:68]: > We have learned the hard way that without such a notice, we will repeatedly be pestered by idiots who think having published this document makes it our job to solve all the world's technical problems.

更改了此處的段落結構以匹配原文表達方式

File: README-zh_CN.md (hunk 1 above)
Comment (ryanhanwu) [reply README-zh_CN.md:64]: Good point!

**Comment by ryanhanwu** [comment]
Thanks for your contribution, well done!

**Pull request merged**